code/game/objects/items/oddities.dm

```diff
@@ -591,3 +591,259 @@
 	)
 	price_tag = 3200 //So we have a reason for stealing it
 	perk = /datum/perk/chemist
+
+//Lonestar
+//Oddities that are only orderable through Lonestar cargo
+
+/obj/item/oddity/ls
+	prob_perk = 0
+
+/obj/item/oddity/ls/collector_coin
+	name = "Collector's Coin"
+	desc = "Got it for a good price!"
+	icon_state = "collector_coin"
+	oddity_stats = list(
+		STAT_COG = 3,
+	)
+
+/obj/item/oddity/ls/pamphlet
+	name = "Old Tourist Pamphlet"
+	desc = "A leaflet with all the great reasons you should visit 'Humanity's Cradle'. A keepsake for those wanting overexposed photos of a past life."
+	icon_state = "earth_pamphlet"
+	oddity_stats = list(
+		STAT_TGH = 3,
+	)
+
+/obj/item/oddity/ls/rod_figure
+	name = "Immovable Rod Action Figure"
+	desc = "More moveable than the real thing; just as stiff."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "immrod"
+	item_state = "nullrod"
+	w_class = ITEM_SIZE_NORMAL
+	oddity_stats = list(
+		STAT_TGH = 3,
+		STAT_MEC = 3,
+	)
+
+/obj/item/oddity/ls/chess_set
+	name = "Chess set"
+	desc = "Pieces - you guessed it - aren't included."
+	icon_state = "chess_set"
+	w_class = ITEM_SIZE_NORMAL
+	oddity_stats = list(
+		STAT_COG = 6,
+	)
+
+/obj/item/oddity/ls/starscope
+	name = "Starscope"
+	desc = "Replica of an ancient, alien device. It was used for sublight, space travel, acting as a 'star compass'."
+	icon_state = "starscope"
+	oddity_stats = list(
+		STAT_COG = 3,
+		STAT_MEC = 6,
+	)
+
+/obj/item/oddity/ls/flashdrive
+	name = "USB flash drive"
+	desc = "An outdated, Earther, data storage device. Little more than a technical curiosity, now.	"
+	icon_state = "flashdrive"
+	oddity_stats = list(
+		STAT_MEC = 3,
+	)
+
+/obj/item/oddity/ls/mutant_tooth
+	name = "Mutated Tooth"
+	desc = "A Cindarite tooth featuring an uncommon (and very painful) mutation. You can't wait to study it!"
+	icon_state = "mutant_tooth"
+	oddity_stats = list(
+		STAT_BIO = 9,
+	)
+
+/obj/item/oddity/ls/manual
+	name = "Specialist Manual"
+	desc = "A rugged, somewhat poor compilation of notes and pages from a variety of books and manuals. The first few pages feature information ranging from how to make a molotov cocktail to how to hotwire a shuttle."
+	icon_state = "manual"
+	oddity_stats = list(
+		STAT_COG = 3,
+		STAT_MEC = 2,
+		STAT_BIO = 2,
+		STAT_TGH = 2,
+		STAT_VIG = 2,
+		STAT_ROB = 2,
+	)
+
+/obj/item/oddity/ls/bouquet
+	name = "Bouquet"
+	desc = "It has the sweetest aroma. You feel the flutter of butterflies."
+	icon_state = "bouquet"
+	item_state = "bouquet"
+	w_class = ITEM_SIZE_NORMAL
+	oddity_stats = list(
+		STAT_TGH = 6,
+	)
+
+/obj/item/oddity/ls/magazine
+	name = "Porn Magazine"
+	desc = "A good, old porn magazine. Nowadays, it's mostly for collection purposes."
```

Comment:
```suggestion
	desc = "A good, swimming magazine. Nowadays, it's mostly for collection purposes."
```

Reply:
Kind of out of theme. If you want to remove the blatant adult reference, maybe something along the lines of:
	name = "Magazine"
	desc = "You bought this for the articles!"